apps/web/src/components/team-builder/v2/builder.module.css

```
@@ -926,199 +972,246 @@
-  gap: 6px;
-  flex-shrink: 0;
-  min-width: 160px;
-  padding: 2px 10px;
-  border-left: 1px solid color-mix(in oklch, var(--border) 60%, transparent);
+:global(.mvline-cell--name) {
+  width: 140px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+:global(.mvline-cell--stat) {
```

Stylelint (17.9.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)